src/patient-registration/patient-registration.scss

```diff
@@ -59,4 +59,24 @@ a.LinkName:hover {
 
 .main {
     background-color: white;
-}
\ No newline at end of file
+}
+:global(.bx--btn--primary){
```

Comment:
If these are global overrides to be applied everywhere then please push them to the carbon overrides in the app shell.

https://github.com/openmrs/openmrs-esm-core/blob/master/packages/esm-styleguide/src/carbon-overrides.css

Reply:
Sure.

Reply:
I am setting up openmrs-esm-core, as soon as I test the css changes, I'll add the changes there and remove from here.
Is this fine to you?

Reply:
I have created a pull request to openmrs-esm-core, as you instructed. 
What should be my next step?